tensorflow/compiler/tf2tensorrt/convert/ops/layer_utils.h

```diff
@@ -269,6 +269,22 @@ class TRTNetworkBuilder {
     return const_layer;
   }
 
+  // Adds a Constant layer that produces a tensor of shape "shape",
+  // type "data_type" and filled with value "scalar".
+  template <typename T>
+  StatusOr<nvinfer1::IConstantLayer*> Constant(
+    const T value, nvinfer1::Dims shape,
+    nvinfer1::DataType data_type) noexcept {
+    StatusOr<TRT_ShapedWeights> const_weights =
+        weight_store_->GetTempWeights(data_type, shape);
+    TRT_ENSURE_OK(const_weights);
+    const_weights->SetValues(value);
```

Comment:
Would you please fix this to handle the return value from the call.
It currently gives this error:
error: ignoring return value of function declared with 'warn_unused_result' attribute [-Werror,-Wunused-result]

Reply:
Done